lib/ui/window/viewport_metrics.h

```diff
@@ -59,6 +63,10 @@ struct ViewportMetrics {
   double physical_view_inset_left = 0;
   double physical_view_inset_front = kUnsetDepth;
   double physical_view_inset_back = kUnsetDepth;
+  double system_gesture_inset_top = 0;
```

Comment:
Keeping with the naming convention used above, these would be `physical_system_gesture_inset_*` or `physical_gesture_inset_*`. I think we should stick to that scheme for consistency. It is also clearer about the fact that those coordinates do not take into account the device pixel ratio.